packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -437,4 +438,24 @@ describe('Type converters:', () => {
             assert.deepStrictEqual(result, showOptions);
         });
     });
+
+    describe.only('#convertCode', () => {
+        it('should return "true" if convertCode(string) is of string type (string)', () => {
```

Comment:
`convertCode` does not return a `boolean` so the description is incorrect:

```suggestion
        it('should convert a "code" of type "string"', () => {
```